src/plugins/intel_cpu/src/config.cpp

```diff
@@ -79,6 +79,13 @@ void Config::readProperties(const std::map<std::string, std::string> &prop, Mode
         if (streamExecutorConfigKeys.end() !=
             std::find(std::begin(streamExecutorConfigKeys), std::end(streamExecutorConfigKeys), key)) {
             streamExecutorConfig.SetConfig(key, val);
+            if (key == ov::affinity.name()) {
+                const auto affinity_val = ov::util::from_string(val, ov::affinity);
+                if (affinity_val == ov::Affinity::CORE) {
```

Comment:
Please process HYBRID_AWARE and NUMA as well

Reply:
Ok, added.